### ISSUE TITLE
Couldn't find official build of vcfvt testware on vsphere60u2

### ISSUE THREAD

**Issue by rogeliosanchez** [vmware/vic#4939]
Build# 9992
vSphere v6.0

```
KEYWORD OperatingSystem . Set Environment Variable GOVC_URL, ${vc-ip}
Documentation:	
Sets an environment variable to a specified value.
Start / End / Elapsed:	20170426 08:23:02.229 / 20170426 08:23:02.231 / 00:00:00.002
08:23:02.231	FAIL	Resolving variable '${vc-ip}' failed: Variable '${vc}' not found.
```

**Comment by rogeliosanchez** [comment]
[5-6-1-VSAN-Simple.zip](https://github.com/vmware/vic/files/962250/5-6-1-VSAN-Simple.zip)

**Comment by rogeliosanchez** [comment]
Seen in build 9992 05/03/17:
[5-6-1-VSAN-Simple.zip](https://github.com/vmware/vic/files/977433/5-6-1-VSAN-Simple.zip)

**Comment by gigawhitlocks** [comment]
found this in the logs :
` 00:00:00.004 VAR ${line} = 2017-05-04 06:52:23 -0700: Exception spawning testbed: RuntimeError: Couldn't find official build of vcfvt testware on vsphere60u2 `

so, basically this happened because of an error in Nimbus.. I'm not entirely sure why it couldn't find the build, but with the Drone 1 hour limitation it's likely we won't have time to retry (assuming a retry would actually work in the first place, which doesn't seem certain to me) since the vcva deploy on Nimbus takes so long.

Thoughts?

**Comment by gigawhitlocks** [comment]
@mhagen-vmware @rajanashok @rogeliosanchez

**Comment by gigawhitlocks** [comment]
Testbed was removed out from under us. Opened a bug with nimbus team 1866952 to see if we can get a `latest` endpoint to hit.

**Comment by mhagen-vmware** [comment]
--build server:release:ob-latest:storage-main

Where you can see that the build string contains 4 portions:

- build product
- build type, can be 'release' or 'beta'
- build number, in your case, you just want 'ob-latest'
- branch

**Comment by mhagen-vmware** [comment]
didn't work, still talking with nimbus folks in bugzilla.

**Comment by jzt** [comment]
[5-3-Enhanced-Linked-Mode.zip](https://github.com/vmware/vic/files/1011200/5-3-Enhanced-Linked-Mode.zip)

**Comment by jzt** [comment]
Seen in [5-6-1-VSAN-Simple.zip](https://github.com/vmware/vic/files/1011314/5-6-1-VSAN-Simple.zip)

**Comment by jzt** [comment]
[5-3-Enhanced-Linked-Mode.zip](https://github.com/vmware/vic/files/1019584/5-3-Enhanced-Linked-Mode.zip)

**Comment by anchal-agrawal** [comment]
Seen in vSphere 6.0 tests: `RuntimeError: Couldn't find official build of vcfvt testware on vsphere60ep8a`
[5-3-Enhanced-Linked-Mode.zip](https://github.com/vmware/vic/files/1030622/5-3-Enhanced-Linked-Mode.zip)
[5-6-1-VSAN-Simple.zip](https://github.com/vmware/vic/files/1030620/5-6-1-VSAN-Simple.zip)
[5-6-2-VSAN-Complex.zip](https://github.com/vmware/vic/files/1030621/5-6-2-VSAN-Complex.zip)
[13-1-vMotion-VCH-Appliance.zip](https://github.com/vmware/vic/files/1030616/13-1-vMotion-VCH-Appliance.zip)
[13-2-vMotion-Container.zip](https://github.com/vmware/vic/files/1030617/13-2-vMotion-Container.zip)

**Comment by anchal-agrawal** [comment]
Seen in vSphere 6.0 tests: `RuntimeError: Couldn't find official build of vcfvt testware on vsphere60ep8a`
[5-3-Enhanced-Linked-Mode.zip](https://github.com/vmware/vic/files/1033746/5-3-Enhanced-Linked-Mode.zip)
[5-6-1-VSAN-Simple.zip](https://github.com/vmware/vic/files/1033747/5-6-1-VSAN-Simple.zip)
[5-6-2-VSAN-Complex.zip](https://github.com/vmware/vic/files/1033743/5-6-2-VSAN-Complex.zip)
[13-1-vMotion-VCH-Appliance.zip](https://github.com/vmware/vic/files/1033745/13-1-vMotion-VCH-Appliance.zip)
[13-2-vMotion-Container.zip](https://github.com/vmware/vic/files/1033744/13-2-vMotion-Container.zip)

**Comment by mhagen-vmware** [comment]
Did that last nightly have my merge in it?

**Comment by anchal-agrawal** [comment]
Not 100% sure, but your commit was on 5/26 at 15:14, and the log bundle is timestamped 5/26 18:30 so it looks like the nightly run should have had your change.

**Comment by anchal-agrawal** [comment]
Seen in vSphere 6.0 tests: `RuntimeError: Couldn't find official build of vcfvt testware on vsphere60ep8a`
[5-3-Enhanced-Linked-Mode.zip](https://github.com/vmware/vic/files/1034397/5-3-Enhanced-Linked-Mode.zip)
[5-6-1-VSAN-Simple.zip](https://github.com/vmware/vic/files/1034400/5-6-1-VSAN-Simple.zip)
[5-6-2-VSAN-Complex.zip](https://github.com/vmware/vic/files/1034401/5-6-2-VSAN-Complex.zip)
[13-1-vMotion-VCH-Appliance.zip](https://github.com/vmware/vic/files/1034399/13-1-vMotion-VCH-Appliance.zip)
[13-2-vMotion-Container.zip](https://github.com/vmware/vic/files/1034398/13-2-vMotion-Container.zip)

**Comment by mhagen-vmware** [comment]
Those were not running the latest code, so let's close this again for now and assume I did fix it :)  Please reopen if we get a run on the latest code and still see this error.

**Comment by jzt** [comment]
Seen in [13-2-vMotion-Container.zip](https://github.com/vmware/vic/files/1350070/13-2-vMotion-Container.zip).

**Comment by mhagen-vmware** [comment]
fixed with #6530
